app/controllers/shopify_app/callback_controller.rb

```diff
@@ -130,8 +130,12 @@ def user_access_scopes_strategy
     end
 
     def perform_post_authenticate_jobs(session)
-      install_webhooks(session)
-      install_scripttags(session)
+      # Ensure we use the offline session to install webhooks and scripttags
+      offline_session = session.online? ? shop_session : session
```

Comment:
i think the name `offline_session` is a bit confusing because I'd assume `shop_session` to be online but then it got assigned to `offline_session`. How about `present_session`?

Reply:
`current_session` might be used already

Reply:
I went with `session_for_shop` as `shop_session` is the right name here, but it was taken 😄